lang/cs/Org.Apache.REEF.Client/YARN/YarnREEFDotNetClient.cs

```diff
@@ -153,6 +133,88 @@ public IJobSubmissionResult SubmitAndGetJobStatus(IJobSubmission jobSubmission)
             throw new NotSupportedException();
         }
 
+        /// <summary>
+        /// Gets the path of the submission package without submitting the job.
+        /// Only available in YarnREEFDotNetClient because implementation is YARN specific.
```

Comment:
If the assertion is that this API should not be added to IREEFClient because this requirement/existence of a packaged version of submission is YARN specific we should discuss a bit more about it.
The package generation part should be put in another interface in that case and YarnREEFDotNetClient should then implement IREEFClient and whatever that other interface is.